.size-snapshot.json

```diff
@@ -1,8 +1,8 @@
 {
   "index.js": {
-    "bundled": 17865,
-    "minified": 8067,
-    "gzipped": 2823,
+    "bundled": 19712,
+    "minified": 8864,
+    "gzipped": 3071,
```

Comment:
I don't know why it increases so much.

Reply:
Sorry, my wrong, Let me fix it.

Reply:
I don't think so, I run quite lately.

Reply:
Wow, that's weird, the same thing, I thought it's my old build problem.